api/swagger.yml

```diff
@@ -1062,6 +1062,12 @@ components:
           type: string
         redirect_uri:
           type: string
+        ttl_seconds:
+          type: integer
+          format: int64
+          description: |
+            The time-to-live for the generated token in seconds.  The maximum
```

Comment:
I'm not sure I understand the second sentence here

Reply:
🤦‍♂️